### PR TITLE
Care to learn

### PR DESCRIPTION
**Pull request by Kittyaoooo**
add care to learn variables

### CODE REVIEW

File: policyengine_uk/parameters/gov/dfe/care_to_learn.py [hunk 1]
```diff
@@ -1,22 +1,18 @@
...
-        # if it's £180 per week, the monthly value is just 180 * 52 / 12
-        live_in_london = household("live_in_london", period)
-        # live_in_london = household("region", period).decode_to_str() == "LONDON"
-        amount_per_child = parameters(period).gov.dwp.universal_credit.elements.childcare.childcare_amount_per_child
-        # amount_per_child = parameters(period).gov.dfe.care_to_learn.amount_per_child
+        live_in_london = household("region", period).decode_to_str() == "LONDON"
+        amount_per_child = parameters(period).gov.dfe.care_to_learn.amount_per_child
         if live_in_london:
-            return amount_per_child.IN_LONDON # multiply to get monthly amount
+            return amount_per_child.IN_LONDON
```
Comment (nikhilwoodruff) [inline policyengine_uk/parameters/gov/dfe/care_to_learn.py:16]: I think this still returns the weekly amount, no?

**Review by nikhilwoodruff — CHANGES_REQUESTED** [review]

**Comment by Kittyaoooo** [comment]
I've multiplied the data in the amount_per_child.yaml to get monthly
amount, so the value returned should be monthly amount.

**Comment by nikhilwoodruff** [comment]
@Kittyaoooo- sorry, we do need to multiply within the formula. This is because when we add references, they need to match the value in the parameter file, and if we've manually changed it to be monthly, it won't match.

File: policyengine_uk/parameters/gov/dfe/care_to_learn.py
Comment (nikhilwoodruff) [inline policyengine_uk/parameters/gov/dfe/care_to_learn.py:1]: This python file should be with the others in the variables/ folder.

File: policyengine_uk/tests/policy/baseline/gov/hmrc/income_tax/allowances/allowance_integration.yaml
Comment (nikhilwoodruff) [inline policyengine_uk/tests/policy/baseline/gov/hmrc/income_tax/allowances/allowance_integration.yaml:1]: These files are from an earlier PR- could you remove them? I'm not sure why they're here.

**Pull request closed without merging**